students/17th/kangminji/js/westalogin.js

```diff
@@ -0,0 +1,65 @@
+const id =document.getElementById('id');
+const password = document.getElementById("password");
+const btn = document.getElementById('btn');
+
+
+
+
+
+
+
+password.addEventListener('keyup', function changcolor(event){
+
+const idvalue = document.getElementById('id').value;
+const passwordvalue = document.getElementById('password').value;
+
+if(idvalue&&passwordvalue){
+btn.style.backgroundColor =  'blue';
+
+}else {
+    btn.style.backgroundColor = ' rgb(185, 216,256)';
+}
```

Comment:
삼항연산자를 활용해 아래와 같이 간편히 작성해주실 수 있습니다. 
```suggestion
idvalue&&passwordvalue ? 
    btn.style.backgroundColor =  'blue' 
    : btn.style.backgroundColor = ' rgb(185, 216,256)'
}
```